app/config/config.yml

```diff
@@ -135,6 +135,25 @@ hwi_oauth:
             options:
                 csrf: true
 
+scheb_two_factor:
+    security_tokens:
+        - Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken
+        - HWI\Bundle\OAuthBundle\Security\Core\Authentication\Token\OAuthToken
+
+    backup_codes:
+        enabled: true
+        manager: Packagist\WebBundle\Security\TwoFactorAuthManager
+
+    totp:
+        enabled: true
+        server_name: '%packagist_host%'
+        issuer: Packagist
+        window: 1
```

Comment:
A new code is generated every 30 seconds.  A `window` of `1` will allow three codes to be accepted:

 - The current code
 - The code generated 30 seconds before it
 - The next code that will be generated

This allows some flexibility in case the server clock and mobile clock are slightly out-of-sync, or the user needs a few extra seconds to enter in their code.